### PR TITLE
Unnamed Features PR #367

### PR DESCRIPTION
**Pull request by 77seven77seven**
## About The Pull Request

yet another strangely quality PR made by ya boi seven7 himself

FEATURES:
- crafting skills are now more easily trainable
- shit falling on people now deals damage (based on how much it weighs)
- people falling on people now deals damage
- NEW ALCOHOL YAY!!!
- bombs are cheaper
- dwarves get buff to mining and blacksmithing, plus the wandering dwarf can now spawn with bonus skills for each
- no banned prayer words for a patron of ZIZO
- thats it
- oh right theres now a counter for how much holes people fall into

## Why It's Good For The Game

GOOD SHIT FEATURES QOL AAAAAGH!! ARGGH!! FALLING SHIT PROMOTES CREATIVITY FOR TRARPS AND DUNGEONS!!! ARGH!!!

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

File: code/modules/jobs/job_types/roguetown/roguetown.dm [hunk 1]
```diff
@@ -49,16 +49,17 @@
 		if(H.dna)
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/roguetown/roguetown.dm:49]: The check right above this is also broken, anyway to fix it?

File: code/modules/jobs/job_types/roguetown/adventurer/types/combat/dbomb.dm [hunk 2]
```diff
@@ -17,9 +17,11 @@
 	backpack_contents = list(/obj/item/bomb = 1, /obj/item/flint = 1)
 	switch(pick(1,2))
 		if (1)
-			beltl = /obj/item/rogueweapon/pick		
+			beltl = /obj/item/rogueweapon/pick
+			H.mind.adjust_skillrank(/datum/skill/labor/mining, 3, TRUE)
 		if (2)
 			beltl = /obj/item/rogueweapon/hammer
+			H.mind.adjust_skillrank(/datum/skill/craft/blacksmithing, 3, TRUE)
```
Comment (ThePainkiller) [inline code/modules/jobs/job_types/roguetown/adventurer/types/combat/dbomb.dm:24]: This class is unused and legacy, unactive, and even if it were, you're out of your mind by making them "blacksmith but better"

**Review by ThePainkiller — CHANGES_REQUESTED** [review]

File: code/modules/jobs/job_types/roguetown/adventurer/types/combat/dbomb.dm (hunk 2 above)
Comment (casualspacestation14enjoyer) [reply code/modules/jobs/job_types/roguetown/adventurer/types/combat/dbomb.dm]: blacksmithing set to 1 and added weaponsmithing

File: code/modules/cargo/packsrogue/food.dm [hunk 3]
```diff
@@ -28,6 +28,11 @@
 	cost = 20
 	contains = /obj/item/reagent_containers/glass/bottle/rogue/beer/blackgoat
 
+/datum/supply_pack/rogue/food/onin
+	name = "Ratkept Onin Cognac"
```
[the remark this answers is not in the document]
Comment (casualspacestation14enjoyer) [reply code/modules/cargo/packsrogue/food.dm:32]: its small lore about rats being the one that kept it safe in a cellar to save it from being stolen by beggars breaking into it

[the remark this answers is not in the document]
Comment (casualspacestation14enjoyer) [reply code/modules/cargo/packsrogue/food.dm:32]: it has been written  on the bottle

File: code/game/objects/structures/fluff.dm [hunk 4]
```diff
@@ -850,6 +850,57 @@
...
+	icon_state = "globe"
+	density = TRUE
+	anchored = FALSE
+
+/obj/structure/fluff/globe/attack_hand(mob/user)
+	if(!ishuman(user))
+		return
+
+	var/mob/living/carbon/human/H = user
+	var/random_message = pick("you spin the globe!", "You land on Rockhill!", "You land on Zybantine!", "You land on port Ice cube!.", "You land on port Thornvale!", "You land on grenzelhoft!")
```
Comment (ThePainkiller) [inline code/game/objects/structures/fluff.dm:901]:
```suggestion
	var/random_message = pick("you spin the globe!", "You land on Rockhill!", "You land on Zybantine!", "You land on port Ice cube!.", "You land on port Thornvale!", "You land on Grenzelhoft!")
```

**Review by ThePainkiller — DISMISSED** [review]

**Review by Stutternov — APPROVED** [review]

**Pull request merged**